apps/examples/testcase/le_tc/kernel/tc_libc_timer.c

```diff
@@ -327,20 +326,20 @@ static void tc_libc_timer_strftime(void)
 	 * time structure has month in range 0-11,
 	 * so tm_mon + 1 represents actual month number */
 
-	strftime(buffer, BUFF_SIZE, "%m", &st_time);
-	TC_ASSERT_EQ("strftime", atoi(buffer), st_time.tm_mon + 1);
+	/* Check the zero value */
+	TC_ASSERT_EQ("strftime", strftime(buffer, 0, "return_0_check", &st_time), 0);
 
-	/* Check the abbreviated month name */
-	strftime(buffer, BUFF_SIZE, "%h", &st_time);
-	TC_ASSERT_EQ("strftime", strcmp(buffer, "Jun"), 0);
+	/* Check the three-letter abbreviation for the day of the week */
+	strftime(buffer, BUFF_SIZE, "a", &st_time);
+	TC_ASSERT_EQ("strftime", strncmp(buffer, "a", 2), 0);
+
+	/* Check the full name for the day of the week */
+	strftime(buffer, BUFF_SIZE, "%a", &st_time);
+	TC_ASSERT_EQ("strftime", strncmp(buffer, "Day", 4), 0);
```

Comment:
Please change strncmp size to calculated number with strlen

Reply:
modify